helm-charts/common/data-prep/templates/deployment.yaml

```diff
@@ -41,11 +41,28 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.entryCommand }}
+          command: {{ .Values.entryCommand }}
+          {{- end }}
+          {{- if .Values.extraArgs }}
+          args:
+            {{- range .Values.extraArgs }}
+            - {{ . | quote }}
+            {{- end }}
+          {{- end }}
           ports:
             - name: data-prep
-              containerPort: {{ .Values.port }}
+              containerPort: {{ .Values.service.containerPort }}
               protocol: TCP
           volumeMounts:
+            {{- if .Values.global.cacheUseHostPath }}
+            - mountPath: /home/user/.cache/clip
+              name: cache-volume
+              subPath: clip
+            - mountPath: /home/user/.cache/huggingface/hub
```

Comment:
The huggingface/hub is for model cache, we can reuse the global.modelUsePVC and modelUseHostPath as https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/common/tgi/templates/deployment.yaml#L83
This way we can support the PVC on multi nodes env.
Example like this:
          volumeMounts:
            - mountPath: /home/user/.cache/huggingface/hub
              name: model-volume

Reply:
Done.